data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1019 @@
+			choice
+				`	"I accept."`
+					goto accepted
+				`	"You want me to do your errands? I won't!"`
+					goto rejected
+			label rude
+				`	"Thy appreciation or lack thereof is immaterial. Thou may'st be an alien, and yet that makes thee even more a guest within our space and even more a, hm... consumer of our hospitality. Thy compliance is expected in return for this generosity.`
+			choice
+				`	"I apologize."`
+				`	"I have no desire to be your servant."`
+					goto rejected
+			`	"Thy apology is noted."`
+				goto task
+			label accepted
+			`	"Thou show'st wisdom in thy choice. Even now, thy ship is prepared within the spaceport of this estate. Thou should'st go there when thy audience is finished, and begin thy task with haste." The scion pauses for a moment before continuing." But note, Captain..." Sieasej shivers with color as the scion speaks the next sentence, "that we do not trust our citizen and servant to thee without consideration. If harm befalls the speaker of these words whilst they are under thy protection, or if thou fail'st to return them here in thirty days, we will consider thee an enemy of this House and levy upon thee every harm that we can offer."`
+				goto end
+			label rejected
+			action
+				set "rejected: Successors: First Contact 2"
+			`	A ripple of color runs up the body of Sieasej and several of the other Successors. The scion gives their response slowly. "Then our act of generosity today shall be allowing thee to depart this noble place whilst still possessed of thy freedom and belongings. Thou shalt receive no further aid nor comfort from this House, and thou shalt be banished at the pleasure of its scions from whatever worlds are held under its dominion."`
+			`	Sieasej and the scions turn their backs to you, and a line of soldiers converge from the walls to form a line separating you from them. The lead officer who escorted you inside motions with their weapon for you to leave.`
+			`	As you exit, the door closes behind you, reforming into solid metal.`
+			label end
+
+substitutions
+	"<tranq h2h>" "Pug staff"
+	"<tranq h2h>" "Heliarch staff"
+		has "outfit: Enforcer Riot Staff"
+
+
+mission "Successors: First Contact 3"
+	priority
+	name "Capture an animal for House Sioeora"
+	description `The scions of House Sioeora have instructed you to visit the planet <planet stopovers> and capture a "Kijra-riitaa" to safely return to them at <planet>. Return by <date> or they will consider you to have kidnapped Sieasej.`
+	source "Staja-Kella-Oa"
+	to offer
+		has "Successors: First Contact 2: done"
+		not "rejected: Successors: First Contact 2"
+	deadline 30
+	stopover "Raaqa-Uur-Kaav"
+	destination "Staja-Kella-Oa"
+	passengers 1
+	blocked `You speak with Sieasej, who informs you that they require at least 1 free bunk to travel.`
+	on offer
+		conversation
```

Comment:
Let me know your thoughts on how I've applied `to display` and condition pruning to these missions. I don't return to the `questions` label for _every_ section because some of the sections have unique follow-up questions; I honestly think it would be more trouble than it's worth to try to implement these in a single master `questions` label because they are designed to be dependent on what was asked immediately prior. 
Nonetheless, this partial strategy does reduce redundancy somewhat.